libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -192,4 +193,26 @@ public static boolean isExpiredFraudVc(VerifiableCredential vc) {
         var now = Instant.now();
         return nbf.plus(expiryPeriod, ChronoUnit.HOURS).isBefore(now);
     }
+
+    public static boolean isFraudCheckUnavailable(List<VerifiableCredential> vcs) {
```

Comment:
See comment above, I did try doing this, but then I was worried about the uncertain case, i.e. where you don't yet know.

It feels more natural at that point to say that the fraud check is 'not unavailable', rather than 'is available'? Semantics perhaps...